tech-notes.md

```diff
@@ -0,0 +1,50 @@
+## Tech Notes
+
+There are 6 canvases on the page:
+
+* `imageCanvas` is the only visible canvas. It's where the composite image is rendered and is downloaded when the user is done.
+* `tempCanvas` holds path data when drawing paths.
+* `holderCanvas` takes a snapshot of the `imageCanvas` as soon as a new path starts.
+* `rotationCanvas`
```

Comment:
What's the purpose of rotationCanvas?

Reply:
I was wondering that too - it looks like it's never actually used, per [this note](https://github.com/SaFrMo/image-scrubber/blob/feature/docs-v2/tech-notes.md#rotating-an-image). I can definitely look into removing unused code like this over the next few days!

Reply:
Oh, its actually used for a few things- both the main blur draw function and to save the canvas-data for the rotation function. My guess is we could take this project's canvas usage down by at least one offscreen canvas, but we'd need to map out the way the image flows between all of them for all the different saves, etc.. it got a bit twisted after a few layers of features and PRs.

Reply:
Oops, sorry, went to edit the readme and did it in your fork @SaFrMo !

Reply:
Haha no worries!